src/pe_reports/_version.py

```diff
@@ -1,2 +1,2 @@
 """Working version of pe-reports."""
```

Comment:
No need to change this from the skeleton:
```suggestion
"""This file defines the version of this module."""
```